packages/ncids-react/lib/index.js

```diff
@@ -0,0 +1,1379 @@
+'use strict';
```

Comment:
What is this and should it really be comitted here?

Reply:
ok, lib is now untracked and it should be published via github action